### ISSUE TITLE
Install Fails: node-sass error

### ISSUE THREAD

**Issue by adammcarth** [bigcommerce/stencil-cli#296]
I've tried installing this on three different machines (2x OSX, 1x Ubuntu Server), and it fails when trying to compile `node-sass`.

```
gyp info spawn args [ 'V=1', 'BUILDTYPE=Release', '-C', 'build' ]
make: Entering directory `/home/adammcarth/.nvm/versions/node/v6.2.1/lib/node_modules/@bigcommerce/stencil-cli/node_modules/node-sass/build'
  g++ '-DNODE_GYP_MODULE_NAME=binding' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DBUILDING_NODE_EXTENSION' -I/home/adammcarth/.node-gyp/6.2.1/include/node -I/home/adammcarth/.node-gyp/6.2.1/src -I/home/adammcarth/.node-gyp/6.2.1/deps/uv/include -I/home/adammcarth/.node-gyp/6.2.1/deps/v8/include -I../../nan  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++0x -std=c++0x -MMD -MF ./Release/.deps/Release/obj.target/binding/src/binding.o.d.raw   -c -o Release/obj.target/binding/src/binding.o ../src/binding.cpp
In file included from ../src/binding.cpp:3:0:
../src/sass_context_wrapper.h:8:26: fatal error: sass/context.h: No such file or directory
 #include <sass/context.h>
                          ^
compilation terminated.
make: *** [Release/obj.target/binding/src/binding.o] Error 1
make: Leaving directory `/home/adammcarth/.nvm/versions/node/v6.2.1/lib/node_modules/@bigcommerce/stencil-cli/node_modules/node-sass/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/adammcarth/.nvm/versions/node/v6.2.1/lib/node_modules/@bigcommerce/stencil-cli/node_modules/node-gyp/lib/build.js:285:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Linux 3.13.0-85-generic
gyp ERR! command "/home/adammcarth/.nvm/versions/node/v6.2.1/bin/node" "/home/adammcarth/.nvm/versions/node/v6.2.1/lib/node_modules/@bigcommerce/stencil-cli/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /home/adammcarth/.nvm/versions/node/v6.2.1/lib/node_modules/@bigcommerce/stencil-cli/node_modules/node-sass
gyp ERR! node -v v6.2.1
gyp ERR! node-gyp -v v3.6.0
gyp ERR! not ok
Build failed
/home/adammcarth/.nvm/versions/node/v6.2.1/lib
└── (empty)

npm WARN optional Skipping failed optional dependency /@bigcommerce/stencil-cli/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.1
npm WARN eslint-config-airbnb@6.2.0 requires a peer of eslint-plugin-react@^4.2.3 but none was installed.
npm ERR! Linux 3.13.0-85-generic
npm ERR! argv "/home/adammcarth/.nvm/versions/node/v6.2.1/bin/node" "/home/adammcarth/.nvm/versions/node/v6.2.1/bin/npm" "install" "-g" "@bigcommerce/stencil-cli"
npm ERR! node v6.2.1
npm ERR! npm  v3.9.3
npm ERR! code ELIFECYCLE

npm ERR! node-sass@3.4.2 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@3.4.2 postinstall script 'node scripts/build.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-sass
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/adammcarth/npm-debug.log
npm ERR! code 1
```

Running `npm install -g node-sass@3.4.2` worked without error on all machines.

**Comment by adammcarth** [comment]
Perhaps it's worth pointing out in the README that `stencil-cli` doesn't install on Node.js >= 6. Would have saved me a couple of hours.

**Comment by adammcarth** [comment]
This was fixed in PR #298 (merged).
